stack_test.go

```diff
@@ -0,0 +1,128 @@
+package errors
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+)
+
+var line, _, _, _ = runtime.Caller(0)
```

Comment:
The variable name `line` is misleading because it actually holds the `pc` value at this point. Consider the name `initpc`.